src/main/java/org/folio/circulation/domain/ServicePoint.java

```diff
@@ -1,56 +1,72 @@
 package org.folio.circulation.domain;
 
-import io.vertx.core.json.JsonObject;
-
-import static org.folio.circulation.support.JsonPropertyFetcher.getProperty;
+import static org.folio.circulation.support.JsonPropertyFetcher.getBooleanProperty;
 import static org.folio.circulation.support.JsonPropertyFetcher.getIntegerProperty;
 import static org.folio.circulation.support.JsonPropertyFetcher.getObjectProperty;
-import static org.folio.circulation.support.JsonPropertyFetcher.getBooleanProperty;
+import static org.folio.circulation.support.JsonPropertyFetcher.getProperty;
+
+import org.joda.time.DateTimeZone;
+
+import io.vertx.core.json.JsonObject;
 
 public class ServicePoint {
   private final JsonObject representation;
-  
+  private DateTimeZone tenantTimeZone;
+
   public ServicePoint(JsonObject representation) {
+    this(representation, null);
+  }
+
+  public ServicePoint(JsonObject representation, DateTimeZone tenantTimeZone) {
```

Comment:
Why does a service point have the tenant wide timezone setting?

Reply:
Main motivation was that probably we have a lot of places where we operate with service point and time zones together.

But will simplify this.